lnwire/node_announcement.go

```diff
@@ -101,13 +96,18 @@ type NodeAnnouncement struct {
 
 	// TODO(roasbeef): add the global features here
 
-	// pad is used to reserve to additional bytes for future
-	// usage.
-	pad uint16
-
-	// Alias is used to customize their node's appearance in maps
-	// and graphs
+	// Alias is used to customize their node's appearance in maps and graphs
 	Alias Alias
+
+	// Features is the list of protocol features this node supports.
+	Features *FeatureVector
+
+	// Address includes two specification fields: 'ipv6' and 'port' on which
+	// the node is accepting incoming connections.
+	Addresses []net.Addr
+
+	// pad is used to reserve to additional bytes for future usage.
+	pad uint16
```

Comment:
Why was this padding added? It isn't a part of the struct. 

You can also remove the `TODO(roasbeef)` above.